src/services/db/TokenService.ts

```diff
@@ -0,0 +1,340 @@
+import { AxiosResponse } from "axios"
+import {
+  Result,
+  ok,
+  err,
+  ResultAsync,
+  okAsync,
+  errAsync,
+  fromPromise,
+} from "neverthrow"
+import { ModelStatic } from "sequelize"
+
+import logger from "@logger/logger"
+
+import DatabaseError from "@errors/DatabaseError"
+import NoAvailableTokenError from "@errors/NoAvailableTokenError"
+import TokenParsingError from "@errors/TokenParsingError"
+
+import { AccessToken } from "@database/models"
+
+// Env vars
+export const GITHUB_TOKEN_LIMIT = 5000
+export const GITHUB_TOKEN_THRESHOLD = 4000 // allowed uses
+export const GITHUB_RESET_INTERVAL = 60 * 60 // seconds
+export const ACTIVE_TOKEN_ALERT_1 = 0.6
```

Comment:
Hmm lets be more intentional in our naming, could we use something like 
`ACTIVE_TOKEN_WARN_LEVEL` and `ACTIVE_TOKEN_ALARM_LEVEL` rather than `alert1` and `alert2`?

Reply:
Make sense!